workflow/rocoto/tasks.py

```diff
@@ -148,11 +155,19 @@ def _get_forecast_hours(cdump, config, component='atmos') -> List[str]:
             fhout = local_config['FHOUT_GFS']
             fhmax_hf = local_config['FHMAX_HF_GFS']
             fhout_hf = local_config['FHOUT_HF_GFS']
-            fhrs_hf = range(fhmin, fhmax_hf + fhout_hf, fhout_hf)
-            fhrs = list(fhrs_hf) + list(range(fhrs_hf[-1] + fhout, fhmax + fhout, fhout))
+
+            sdate = config['SDATE']
+            cyc = sdate.hour
+            offset = (cyc + fhmin) % fhout
+            if component in 'ice' and offset != 0:
+                fhrs_hf = range(fhmin + 12, fhmax_hf + fhout_hf, fhout_hf)
+                fhrs = list(fhrs_hf) + list(range(fhrs_hf[-1] + fhout, fhmax, fhout))
+            else:
+                fhrs_hf = range(fhmin, fhmax_hf + fhout_hf, fhout_hf)
+                fhrs = list(fhrs_hf) + list(range(fhrs_hf[-1] + fhout, fhmax + fhout, fhout))
```

Comment:
@WalterKolczynski-NOAA A routine has been added to adjust `fhrs` in cases where the `component` is `ice` and the `offset `is not `0`. The main challenge here is to find a `cyc` variable. The best I could find is `sdate.hour`, but this may be problematic if there are multiple different `sdate.hour`'s between `sdate `and `edate`.

Reply:
Sure. I brought back the changes I had before. I think we can figure out a better way to do this in a separate PR.